### PR TITLE
Model selection

### PR DESCRIPTION
**Pull request by Ganling6666**
Trying to select the best fit model for each demographic identifier.

### CODE REVIEW

File: processing-and-analysis/02-command-files/Ganling's file/Random_effect_Interaction.Rmd [hunk 1]
```diff
@@ -152,10 +152,102 @@ df
...
+# remove experiment1 * dem_id from the best model predicted 
+urm_mod3 <- lmer(finalexam ~ experiment1 + urm_id + satmath + satverbal + mastered_topics_initial_kc +  high_sch_gpa + (1 | ta_sect), data = master_true, REML = FALSE)
+
+AIC(fgn_mod2, fgn_mod3)
+
+#Refit model 3 with lmer and REML = FALSE
+urm_mod4 <- lmer(finalexam ~ experiment1 + urm_id + satmath + satverbal + mastered_topics_initial_kc +  high_sch_gpa + (1 | ta_sect), data = master_true, REML = TRUE)
+
+AIC(urm_mod3, urm_mod4)
+```
```
Comment (cfcraig) [inline processing-and-analysis/02-command-files/Ganling's file/Random_effect_Interaction.Rmd:243]: Please add `summary(_dem__mod4)` for each demographic variable

**Review by cfcraig — CHANGES_REQUESTED** [review]
Please add `summary(_dem__mod4)` for each demographic variable in the model selection section.

**Comment by Ganling6666** [comment]
Made some changes. Now I have correct data-wrangling codes but the model selection part is not making sense.

**Review by cfcraig — CHANGES_REQUESTED** [review]
* Complete the model selection with correct dataset
* Update script according to [style guide](https://docs.google.com/document/d/1sycWoBUs5jPM83M2ywRJtv-EBo1t2rvR1wDhM7jS1W8/edit#heading=h.piug88d0vy62) introduced in 2021-08-10 meeting.

**Comment by Ganling6666** [comment]
The new file for model selection is called "random_effect_model_selection". I created a new file to make it look less messy

**Review by cfcraig — APPROVED** [review]
Good use of commits to structure your model selection...it was easy to follow your changes. Couple things I'd also like to see in the future:

* More descriptive commit messages, so I have a better sense of the purpose of the changes in each commit
* Commit messages that are formatted as an action, such as: "conduct model selection on `fgn` and `urm`" for your last commit.

**Pull request merged**